batch/indexer_Position_Coupon.py

```diff
@@ -176,22 +175,37 @@ def __get_contract_list(self, db_session: Session):
         # Remove duplicate exchanges from a list
         self.exchange_address_list = list(set(_exchange_list_tmp))
 
-    def __sync_all(self, db_session: Session, block_from: int, block_to: int):
-        LOG.info("syncing from={}, to={}".format(block_from, block_to))
-        self.__sync_transfer(db_session, block_from, block_to)
-        self.__sync_consume(db_session, block_from, block_to)
-        self.__sync_exchange(db_session, block_from, block_to)
-        self.__sync_escrow(db_session, block_from, block_to)
+    def __sync_all(self, db_session: Session, block_to: int):
+        sync_from = self.__get_oldest_cursor(self.token_list, block_to)
+        LOG.info("syncing from={}, to={}".format(sync_from, block_to))
+
+        self.__sync_transfer(db_session, block_to)
+        self.__sync_exchange(db_session, block_to)
+        self.__sync_escrow(db_session, block_to)
+
+        self.__update_cursor(block_to + 1)
```

Comment:
After all sync method is done, update the block number where processor should start next.